Chart.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: haystack
+description: A Helm chart for haystack
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
+
+
+# This is the list of dependencies that are required for the chart to work.
+dependencies:
+  - name: elasticsearch
+    version: 7.16.3
```

Comment:
We should switch to a version before the license change (https://www.elastic.co/pricing/faq/licensing) until haystack uses a higher version too. So <=7.10.

Reply:
👍🏻 Makes sense I am going to change it.